detect_regressions.py

```diff
@@ -52,7 +60,7 @@ def detect_regressions(database_file):
                         category = df_test.category.unique()[0]
                         metric_threshold = (
                             df_test[metric][-13:-3].mean()
-                            + 2 * df_test[metric][-13:-3].std()
+                            + 1 * df_test[metric][-13:-3].std()
```

Comment:
I'd still advocate for 2 standard deviations: a ~40% false positive rate would get annoying pretty quickly

Reply:
Yes I agree, I was just trying to get regressions on purpose to see things work, I'll revert this back soon.